src/bootstrap.js

```diff
@@ -230,8 +230,24 @@ import Floom, { Input, Viewport, CombinedRenderer, Vector2, Debug, Tool } from "
 	// create obstacles
 	// fluidSystem.doObstacles = true;
 	fluidSystem.obstacles.push(
-		new Floom.Obstacle(-20, 20, 5),
-		new Floom.Obstacle( 20,  0, 9)
+		new Floom.Obstacle({
+			type: 'circle',
```

Comment:
Having a type attribute is always a red flag for incorrect composition. simply using polymorphism such suffice.

Reply:
replcaced to
```js
new Floom.obstacles.Circle(-20, 20, 5),
new Floom.obstacles.Circle(20, 0, 9),
new Floom.obstacles.Capsule(-5, 0, 15, 20, 5)
```